db/migrate/20180208170522_rename_dimensions_items_content_json_to_raw_json.rb

```diff
@@ -0,0 +1,5 @@
+class RenameDimensionsItemsContentJsonToRawJson < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :dimensions_items, :content_json, :raw_json
```

Comment:
Instead of adding another migration in the same PR to rename a previous column name, it feels better to just rename the field in the first commit.
What about fixing up all commits into a single commit?